packages/bpk-component-barchart/src/BpkBarchart.js

```diff
@@ -45,17 +47,88 @@ const getClassName = cssModules(STYLES);
 const spacing = remToPx(spacingXs);
 const lineHeight = remToPx(lineHeightSm);
 
-const getMaxYValue = (dataPoints, yScaleDataKey, outlierPercentage) => {
+const getMaxYValue = (dataPoints: Function, outlierPercentage: ?number) => {
   const meanValue = dataPoints.reduce((d, t) => d + t, 0) / dataPoints.length;
   const maxYValue = Math.max(...dataPoints);
 
-  return outlierPercentage !== null
+  return outlierPercentage
     ? Math.min(maxYValue, meanValue * (outlierPercentage / 100) + meanValue)
     : maxYValue;
 };
 
-class BpkBarchart extends Component {
-  constructor(props) {
+type Props = {
+  data: any,
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?

Reply:
Yeah I had a look at this but data as a propType is defined as `dataProp` which when I had a look at what it returns and does it seems to be a mixed object of `Error` or `null` that returns and doesn't seem to have a straight forward type I tried a few different types for this prop but it would cause errors in other places in the chain and the component which caused it to be incompatible